#! /bin/sh /usr/share/dpatch/dpatch-run
## 20-Makefile-fPIC.dpatch by Marc Haber <mh+debian-packages@zugschlus.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Build with -fPIC as appropriate for a shared library

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./Makefile /tmp/dpep-work.8ZBFLg/snoopy-1.3/Makefile
--- ./Makefile	2005-08-10 13:21:56.000000000 +0000
+++ /tmp/dpep-work.8ZBFLg/snoopy-1.3/Makefile	2005-08-10 13:33:13.000000000 +0000
@@ -3,13 +3,14 @@
 
 CC = gcc
 LIBS  = -ldl
+CFLAGS = -O2 -W -Wall -g
 
 all: snoopy.so detect
 
 snoopy.so:  snoopy.c snoopy.h
-			$(CC) -shared -O3 -fomit-frame-pointer snoopy.c -osnoopy.so $(LIBS)
+			$(CC) $(CFLAGS) -shared -fPIC -fomit-frame-pointer snoopy.c -osnoopy.so $(LIBS)
 detect: detect.c
-			$(CC) detect.c -odetect $(LIBS)
+			$(CC) $(CFLAGS) detect.c -odetect $(LIBS)
 install: all 
 			install -m 755 snoopy.so /lib/snoopy.so; \
 			./install.sh
